lib/github.com/athas/vector/vspace.fut

```diff
@@ -39,6 +39,12 @@ module type vspace = {
 
   -- | Transform to unit vectortor.
   val normalise: vector -> vector
+
+  -- | Neutral element for the addition.
+  val zero: vector
+
+  -- | Neutral element for the multiplication.
+  val one: real
```

Comment:
Why is this one not a vector?  Shouldn't we have a neutral element for vector `*` as well?

Reply:
Actually it's the multiplication with a scalar (which is the only multiplication that needs to be defined in a vector space).

Reply:
Is that useful?  When do you have a `vector` module without also having the module containing the scalars?

Reply:
Actually I didn't use because I'm lazy but it could be useful for normalize a/b = 1/b * a. Maybe it's not ultra useful and could be in scalar instead. Although I'm not really sure it really goes there neither.